charmtools/build/tactics.py

```diff
@@ -470,19 +470,29 @@ class MetadataYAML(YAMLTactic):
     """Rule Driven metadata.yaml generation"""
     section = "metadata"
     FILENAME = "metadata.yaml"
-    KEY_ORDER = ["name", "summary", "maintainer",
-                 "description", "tags",
-                 "requires", "provides", "peers"]
+    KEY_ORDER = ["name",
+                 "summary",
+                 "maintainer",
+                 "maintainers",
+                 "description",
+                 "tags",
+                 "requires",
+                 "provides",
+                 "peers"]
```

Comment:
nitpick but I'd like to see ] at the same level of `KEY_ORDER` and `"peers"` end in a `,`. This way we can grow the list without having "peers" appear to change just because we added an element